### PR TITLE
Add keycloak admin username and password to the che deployment

### PR DESCRIPTION
**Pull request by vparfonov**
Signed-off-by: Vitalii Parfonov <vparfono@redhat.com>

### CODE REVIEW

File: pkg/deploy/deployment_che.go [hunk 1]
```diff
@@ -130,6 +130,59 @@ func getSpecCheDeployment(deployContext *DeployContext, cmResourceVersion string
...
+			ValueFrom: &corev1.EnvVarSource{
+				SecretKeyRef: &corev1.SecretKeySelector{
+					Key: "password",
+					LocalObjectReference: corev1.LocalObjectReference{
+						Name: identityProviderSecret,
+					},
+				},
+			},
+		},
+			corev1.EnvVar{
```
Comment (tolusha) [inline pkg/deploy/deployment_che.go:151]: pls, format the lines

File: pkg/deploy/deployment_che.go [hunk 2]
```diff
@@ -130,6 +130,59 @@ func getSpecCheDeployment(deployContext *DeployContext, cmResourceVersion string
...
+						},
+					},
+				},
+			})
+	} else {
+		cheEnv = append(cheEnv, corev1.EnvVar{
+			Name:  "CHE_KEYCLOAK_ADMIN__PASSWORD",
+			Value: deployContext.CheCluster.Spec.Auth.IdentityProviderPassword,
+		},
+			corev1.EnvVar{
```
Comment (tolusha) [inline pkg/deploy/deployment_che.go:167]: format the lines

File: pkg/deploy/deployment_che.go (hunk 1 above)
Comment (vparfonov) [reply pkg/deploy/deployment_che.go:151]: Looks, like it well formatted file (with `gofmt`)

File: pkg/deploy/deployment_che.go (hunk 2 above)
Comment (vparfonov) [reply pkg/deploy/deployment_che.go:167]: Looks, like this is well formatted file (with `gofmt`)

**Review by tolusha — APPROVED** [review]

**Pull request merged**